web/src/frontend/src/app/components/underskjema/underskjema.less

```diff
@@ -0,0 +1,42 @@
+@import "../../styles/app";
+
+@bakgrunn: #efefef;
+@ramme:    @navGra20;
+
+.underskjema {
+  display: block;
+  padding-top: 0.8rem;
+  padding-bottom: 1rem;
+
+  &__boks {
+    display: block;
+    border: 1px solid @ramme;
+    background-color: @bakgrunn;
+    width: 100%;
+    padding-left: @skjemaMarg / 2;
+    padding-right: 2rem;
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+    position: relative;
+    z-index: 2;
+  }
+
+  &__innhold:before {
+    content: '';
+    height: 1rem;
+    width: 1rem;
+    background-color: @bakgrunn;
+    display: block;
+    position: absolute;
+    transform: rotate(-45deg);
+    transform-origin: 0 0;
+    border-top: 1px solid @ramme;
+    border-right: 1px solid @ramme;
+    box-sizing: border-box;
```

Comment:
Klarer vi å sette denne globalt, eller må vi gjøre det for alle komponenter? Jeg vet ikke hvordan nav-frontend-moduler gjør dette, men det hadde vært fint å hatt en selector som gjorde at vi slapp å legge til denne. Men vet ikke hvordan ting er gjort før, så kun en tanke